src/image/Image.js

```diff
@@ -51,7 +51,8 @@ class Image extends React.Component {
     } = this.props;
 
     const hasImage =
-      Boolean(attributes.source) && Boolean(attributes.source.uri);
+      typeof attributes.source === 'number' ||
```

Comment:
here is checking for the case `ImageRequireSource` of Image from react-native

Reply:
Can you explain the type `number` used here?
Source of the image can be a string || object :thinking:

Reply:
Yes, I can @pranshuchittora. So in react native source will have type `ImageSourcePropType` like the image above
<img width="515" alt="index_d_ts_—_dentsu" src="https://user-images.githubusercontent.com/30365498/98187029-5d1f2180-1f53-11eb-9a0f-246a8771d720.png">

and this `ImageSourcePropType` will be a union of `ImageURISource | ImageURISource[] | ImageRequireSource;`
<img width="634" alt="index_d_ts_—_dentsu" src="https://user-images.githubusercontent.com/30365498/98187155-a1122680-1f53-11eb-9347-9f50fb10c35c.png">

and `ImageRequireSource` is number. 
<img width="599" alt="index_d_ts_—_dentsu" src="https://user-images.githubusercontent.com/30365498/98187232-cdc63e00-1f53-11eb-87fe-ac1930638f52.png">

Since the source type comes from react-native offical so I think we should also support for ImageRequireSource otherwise, it seems like we are just dropping a part from it

Reply:
@pranshuchittora could you please check this out to see does it make sense to you

Reply:
Hi, I just closed the issue related to this PR. I think there's no issue at all, the usage was wrong pls refer the closing comment. :)

Reply:
uhm, I am quite sad that you say that, since all the images I put above is clear enough for this case. The `source` for `Image` in `react-native-elements` got the type `ImageRequireSource` but seems like now we just dont allow user to use it anymore.